### ISSUE TITLE
How to get inference result?

### ISSUE THREAD

**Issue by TNA8** [QData/spacetimeformer#19]
Thanks for your great work.

I want to infer a trained spacetimeformer model. The model needs 4 inputs; `(x_c, y_c, x_t, y_t)`.
We have only `x_c`, `y_c`, and `x_t` ready.
* `y_t` must be zeros with right shape. Right?
* `output, (logits, labels), attn = spacetimeformer(x_c, y_c, x_t, y_t)`.  The `output` has two tensors; `output.loc` and `output.scale`.
  What is the real output for `y_t` here? `output.loc` or `output.scale`?
  
  Thanks in advance.

**Comment by jakegrigsby** [comment]
Hi, if you are using a MSE or MAE loss function, the output you are looking for is `output.loc`, which is the mean of the distribution the model predicts. `output.scale` is the std dev and it will not be informative unless you are training with the NLL loss function.

**Comment by TNA8** [comment]
We must feed zero `y_t` to the model. Right?

**Comment by TNA8** [comment]
And the `output.loc` needs inverse scaling or can be used directly as predicted values?

**Comment by jakegrigsby** [comment]
Hopefully this is fixed with the new `Forecaster.predict` method
